setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import find_packages, setup
+
+setup(
+    name="turbot",
+    version="2.0.0",
+    author="TheAstropath",
+    license="MIT",
+    packages=find_packages(),
+    long_description=open("README.md", "r").read(),
+    entry_points={
+        "console_scripts": ["turbot=turbot:main"],
```

Comment:
console_scripts has to be just one list containing all the entry points (both turbot and migrate).  As it is right now, it only installs the migrate script, since the first list containing turbot gets replaced with the second one containing migrate